lib/index.css

```
@@ -16,11 +16,151 @@
 
 @import url(lib/bike-rental.css);
 
+@import url(react-sliding-pane/dist/react-sliding-pane.css);
+
+
 /* Hide IE/Edge clear button in text input fields. */
 input[type=text]::-ms-clear {
     display: none;
 }
 
+/* New app menu */
+.app-menu-icon {
+  width: 21px;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  cursor: pointer;
+  position: absolute;
+  z-index: 10;
+  transition: all 1s ease;
+  top: 16px;
+}
+
+.menu-top-line, 
+.menu-middle-line, 
+.menu-bottom-line,
+.menu-left-x,
+.menu-right-x {
+  border-bottom: 3px solid #ffffff;
+  transition: all 0.5s ease;
+}
+
+.menu-left-x {
+  transform: rotate(45deg);
+  top: 7px;
+  position: relative;
+}
+
+.menu-right-x {
+  transform: rotate(-45deg);
+  bottom: 5px;
+  position: relative;
+}
+
+.slide-pane {
+  transition: transform 0.4s;
+}
+
+.slide-pane_from_left {
+  margin: 52px auto 0 0;
+}
+
+.slide-pane__header,
+.slide-pane__close {
+  display: none;
+}
+
+.slide-pane__content {
+  padding: 24px 0;
+}
+
+.app-menu {
+  margin: 0;
+  padding: 0;
+  font-size: 1.3rem;
+}
+
+.app-menu li {
+  padding: 1rem 2rem;
```

```suggestion
  padding: 0.75rem 2rem;
```
I would tighten this up a bit

---

Insert `font-size: 20px` to make the text larger.

---

I would make horiz padding 0.5rem, but your call.